google-cloud-logging/src/main/java/com/google/cloud/logging/LoggingOptions.java

```diff
@@ -108,6 +118,8 @@ public LoggingOptions build() {
   protected LoggingOptions(Builder builder) {
     super(LoggingFactory.class, LoggingRpcFactory.class, builder, new LoggingDefaults());
     this.autoPopulateMetadataOnWrite = builder.autoPopulateMetadataOnWrite;
+    this.batchingSettings =
+        builder.batchingSettings == null ? null : builder.batchingSettings.toBuilder().build();
```

Comment:
Is the call `batchingSettings.toBuilder().build()` necessary here? In the builder code (line 107) this transformation is already executed.

Reply:
Oh, good catch. I believe I will leave constructor as is and remove it from `setBatchingSettings()` - this way it is more clear